src/module.cc

```diff
@@ -78,7 +78,7 @@ NAN_METHOD(GetReport) {
   std::ostringstream out;
 
   GetNodeReport(isolate, kJavaScript, "JavaScript API", __func__, out);
-  // Return value is the NodeReport filename
+  // Return value is the contents of a NodeReport as a string.
```

Comment:
Suggestion: Since this is a newly introduced line, replace "NodeReport" with "report" so it doesn't need to change again for #52.